src/test/java/api/support/builders/RequestPolicyBuilder.java

```diff
@@ -0,0 +1,57 @@
+package api.support.builders;
+
+import java.util.UUID;
+
+import io.vertx.core.json.JsonObject;
+
+public class RequestPolicyBuilder extends JsonBuilder implements Builder {
+  private final UUID id;
+  private final String name;
+  private final String description;
+  private final boolean allowHold;
+  private final boolean allowPage;
+  private final boolean allowRecal;
+
+  public RequestPolicyBuilder() {
+    this(UUID.randomUUID(),
+      "Example Request Policy",
+      "An example request policy",
+      false,
+      false,
+      false
+    );
+  }
+
+  private RequestPolicyBuilder(
+    UUID id,
+    String name,
+    String description,
+    boolean allowHold,
+    boolean allowPage,
+    boolean allowRecal) {
+
+    this.id = id;
+    this.name = name;
+    this.description = description;
+    this.allowHold = allowHold;
+    this.allowPage = allowPage;
+    this.allowRecal = allowRecal;
+  }
+
+  @Override
+  public JsonObject create() {
+    JsonObject request = new JsonObject();
+
+    if (id != null) {
+      put(request, "id", id.toString());
+    }
+
+    put(request, "name", this.name);
+    put(request, "description", this.description);
+    put(request, "allowHold", this.allowHold);
```

Comment:
Please can this be changed to match the request policy record structure described by [request-policy-storage](https://github.com/folio-org/mod-circulation-storage/blob/master/ramls/request-policy.json)

Reply:
Corrected in https://github.com/folio-org/mod-circulation/pull/182/commits/05a744fb8a2c8205bf74786c98bb0a41cd18cf08